### PR TITLE
Explain Staking Targets Table in Polkadot-JS App

### PR DESCRIPTION
**Pull request by alfarok**
# Explain Staking Targets Table in Polkadot-JS App

### Task
This PR addresses #3240 and was ported from PR #3412 which was created on a fork.

### Acceptance Criteria

- [x] Add descriptions of what all the [columns](https://polkadot.js.org/apps/#/staking/targets) mean in the Targets table:
    - [x] `payout`
    - [x] `nominators` (active and all)
        - [x] explanation for each
        - [x] example for each
    - [x] `comm.`
    - [x] `total stake`
    - [x] `own stake`
    - [ ] ~~`other stake`~~
    - [x] `return`
- [x] New information is added or linked to the following existing resources:
    - [x] [maintain-guides-how-to-nominate-polkadot](https://wiki.polkadot.network/docs/maintain-guides-how-to-nominate-polkadot)
    - [x] [learn-nominator](https://wiki.polkadot.network/docs/learn-nominator)
    - [ ] ~~[how-do-i-know-which-validators-to-choose](https://support.polkadot.network/support/solutions/articles/65000150130-how-do-i-know-which-validators-to-choose-)~~
- [x] Screenshots/imagery is updated on existing resources (if required)

### Questions

1. `other stake` is mentioned in original task but I no longer see this option in the chart header.  Was it possibly removed or am I missing something?

2. Where is the code for `support.polkadot.network` hosted?

3. If it is not too difficult we could follow-up with adding or requesting to **add the Target column definitions as tooltips when hovered**?  Difficulty likely depends on how much effort is required for localization.

### CODE REVIEW

**Comment by alfarok** [comment]
Thanks @DrW3RK, opening the pull request on the main repo as opposed to a fork solved the CICD issue.  This may be something we can add to the contributing guide.  I can imagine it being an issue for 3rd party contributions in the future.  I personally tend to fork repos to experiment with changes and open PRs directly from my forks.  It feels less likely that I would accidentally push to the wrong location and disturb anything in prod but it is personal preference.

Also I had to make sure I had the `admin:org` permission on my GitHub access token when pushing to the main repo from the cli.

Feel free to merge if you think this is ready to go.

File: docs/learn/learn-nominator.md [hunk 1]
```diff
@@ -219,13 +219,53 @@ more information and instructions for claiming rewards.
...
+
+- **payout**: How recently the validator has made it's last reward payout to nominators.
+- **nominators**: This column consists of two number values.
+
+  1) Amount of nominators currently bonded in the current era and considered `active`.
+
+  2) Total amount of nominators that nominated that validator.
+
+  - You may want to be cautious of validators with a high number of subscribers. A validator is
+    considered oversubscribed when more than 256 nominators have nominated the validator. In this
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
    considered oversubscribed when more than 256 'active' nominators are assigned to the validator. In this
```

File: docs/learn/learn-nominator.md [hunk 2]
```diff
@@ -219,13 +219,53 @@ more information and instructions for claiming rewards.
...
+    scenario only the top 256 nominators will receive rewards. The remaining subscribers will
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
    scenario only the top 256 nominators will receive rewards. The remaining nominators will
```

File: docs/learn/learn-nominator.md [hunk 3]
```diff
@@ -219,13 +219,53 @@ more information and instructions for claiming rewards.
...
+    recieve nothing, however they can also be slashed in the event that the validator commits a
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
    recieve nothing, however they can be slashed in the event that validator commits a
```

File: docs/learn/learn-nominator.md [hunk 4]
```diff
@@ -219,13 +219,53 @@ more information and instructions for claiming rewards.
...
+    slashable offence.
+  - Every nominator can select up to a maxium of 16 validators, which contributes towards maximizing
+    the probability of having the nominators stake applied to the validators active set. Having too
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
    the probability of having the nominators stake applied to the validators active set. Nominating too
```

File: docs/learn/learn-nominator.md [hunk 5]
```diff
@@ -219,13 +219,53 @@ more information and instructions for claiming rewards.
...
+    few validators could results in the nominator being excluded from the active set. This behaviour
+    is a result of the election algorithm attempting to maximize the overall network stake, while
+    minimizing the variance of the active stake across the validators. For additional information on
+    the selection process checkout the research behind
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
    few validators could result in the nominators losing their rewards when none of them make it to active set or 
    when those Validator nodes stop validating. The election algorithm attempts to maximize the overall network 
    stake, while minimizing the variance of the active stake across the validators. For additional information on
    the election process checkout the research behind
```

**Review by DrW3RK — APPROVED** [review]
minor typo fixes, uniformity of terminology and slight refinement of the content. Feel fee to commit the suggestions directly or refine the suggestions further

**Pull request merged**